### DISCUSSION TITLE
Microstructural evolution of multi-component and multi-phase steel

### DISCUSSION THREAD

**Discussion by aminsakic** [idaholab/moose#20732] [Q&A Modules: Phase field] [answered]
Hello everyone,

I started studying the phase field method and there are some things that I am confused about.
So, first i want to reproduce the results from the paper https://www.sciencedirect.com/science/article/pii/S0927025615007053?casa_token=qMFeBGAUOtYAAAAA:1h1ybuqIpB-BVNrFld5PLev3m4h_8ekojyxLKndauo_uqAhKJcVbOGy3i_96LX63QNdhVs3rjwA.

And I'm already having a hard time getting an initial polycrystalline microstructure like in Figures 3a and 3b. Can this be achieved with the PolycrystalVoronoi object?

Thank you for reading my post, I would really appreciate your answers.

Best regards,
Amin

**Comment by GiudGiud** [comment]
@SudiptaBiswas @laagesen any thoughts on this?

Guillaume

**Comment by SudiptaBiswas** [comment, marked as answer]
The 3D microstructure in the linked paper (3b) looks like a typical Voronoi structure. You should be able to generate something similar with PolycrystalVoronoi objects. See initial condition examples in MOOSE. 

The 2D microstructure seems to have curved grain boundaries, the Voronoi initial condition would not create that (it only creates straight boundaries). You will have to include some sort of anisotropy and then let the structure evolve to such a state. 

Note: In either of these cases, reproducing the exact microstructure might not be feasible without knowing the grain centers.
